projects/showcase/cypress/e2e/terrain-control.cy.ts

```diff
@@ -1,49 +1,50 @@
-import { E2eDriver } from '../support/e2e-driver';
+import { E2eDriver } from "../support/e2e-driver";
 
-describe('Terrain Control', () => {
+describe("Terrain Control", () => {
   context(
-    'Given I am on the Terrain Control showcase and Terrain is disabled',
+    "Given I am on the Terrain Control showcase and Terrain is disabled",
     () => {
-      let driver = new E2eDriver();
+      let { beforeAndAfter, when, get, then } = new E2eDriver();
+      let initialImageSnapshot: any;
+
+      beforeAndAfter();
 
       beforeEach(() => {
-        driver
-          .visitMapPage('/demo/terrain-control')
-          .waitForMapToIdle()
-          .assert.mapTerrainButtonIsOff()
-          .takeImageSnapshot();
+        ({ when, get, then } = new E2eDriver());
```

Comment:
This line and line 7 seems identical, what am I missing?
This repeats in other tests as well.

Reply:
This is recreating the Driver before each test. It is to make sure there are no side effects left between the execution of tests.

Reply:
This sounds like a smell. The beforeEach and afterEach should take care of it.